src/cli/cluster/worker.js

```diff
@@ -82,8 +85,15 @@ module.exports = class Worker extends EventEmitter {
   }
 
   onMessage(msg) {
```

Comment:
I suppose you would need to do some optional param checking since this function technically supports being passed non-arrays.